### PR TITLE
 Copy updates to new pricing page.

### PR DESCRIPTION
**Pull request by mbrandonw**
Most of the copy changes we discussed yesterday.

### CODE REVIEW

File: Sources/Models/Episode.swift [hunk 1]
```diff
@@ -93,7 +93,7 @@ public struct Episode {
     }
   }
 
-  public enum Permission {
+  public enum Permission: Equatable {
```
Comment (mbrandonw) [inline Sources/Models/Episode.swift]: needed this to count the # of free episodes

File: Sources/PointFree/SiteMiddleware.swift [hunk 2]
```diff
@@ -173,7 +174,18 @@ private func render(conn: Conn<StatusLineOpen, T3<(Models.Subscription, Enterpri
         |> pricingResponse
 
     case .pricingLanding:
-      return conn.map(const(user .*. subscriberState .*. route .*. unit))
+      let allEpisodeCount = AllEpisodeCount(rawValue: Current.episodes().count)
+      let episodeHourCount = EpisodeHourCount(rawValue: Current.episodes().reduce(0) { $0 + $1.length } / 3600)
+      let freeEpisodeCount = FreeEpisodeCount(rawValue: Current.episodes().lazy.filter { $0.permission == .free }.count)
```
Comment (mbrandonw) [inline Sources/PointFree/SiteMiddleware.swift]: a bunch of tagged types to represent these integer values...

File: Sources/Views/PricingLanding.swift [hunk 3]
```diff
@@ -134,81 +150,88 @@ private let contactusButtonClasses =
     | Class.border.all
     | Class.pf.colors.border.gray800
 
-private let plansAndPricing = [
```
Comment (mbrandonw) [inline Sources/Views/PricingLanding.swift]: lots of churn in this file due to re-indenting some stuff. you may want to hide white space changes in this PR review

File: Sources/Views/PricingLanding.swift [hunk 4]
```diff
@@ -724,39 +692,68 @@ private func footer(currentUser: User?) -> [Node] {
...
-    <> lightBottomBorder % key("border-bottom", "1px solid #e8e8e8")
+  static func personal(
+    allEpisodeCount: AllEpisodeCount,
+    episodeHourCount: EpisodeHourCount
+    ) -> PricingPlan {
+    return PricingPlan(
+      cost: Cost(title: "per&nbsp;month, billed&nbsp;annually", value: "$14"),
+      features: [
+        "All \(allEpisodeCount.rawValue) episodes with transcripts",
+        "Over \(episodeHourCount.rawValue) hours of video",
```
Comment (mbrandonw) [inline Sources/Views/PricingLanding.swift]: I went with hours. "2,017 minutes" didn't sound as good

File: Sources/PointFree/PricingLanding.swift [hunk 5]
```diff
@@ -11,20 +11,20 @@ import Views
 public let pricingLanding: Middleware<
   StatusLineOpen,
   ResponseEnded,
-  Tuple3<User?, SubscriberState, Route>,
+  Tuple6<User?, AllEpisodeCount, EpisodeHourCount, FreeEpisodeCount, Route, SubscriberState>,
```
Comment (stephencelis) [inline Sources/PointFree/PricingLanding.swift]: Ha lotta changes.

**Pull request merged**